seed/ts-sdk/errors/src/api/resources/simple/client/Client.ts

```diff
@@ -68,45 +72,12 @@ export class Simple {
 
         if (_response.error.reason === "status-code") {
```

Comment:
Great change, however, if the switch only has a `default` case, simply omit the switch and execute the contents of the default case directly.